.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# .github/ is restricted
+.github/ @ryscheng
```

Comment:
this should be a team--not a (particularly busy) person

Reply:
@nhynes teams wasn't working for whatever reason. will have to figure that out later

Reply:
can we just not impose this change then? it's like granting "emergency powers" in wartime and having them never be retracted

Reply:
actually we figured out the problem 😄 we think... stand by